src/algorithms/model.ts

```diff
@@ -556,19 +556,33 @@ export function collectTotals(trajectory: SimulationTimePoint[], ages: string[])
   return res
 }
 
-export function exportSimulation(result: UserResult) {
+export function exportSimulation(result: UserResult, ...ageGroups: string[]) {
```

Comment:
It seem that the variable arguments are not needed here, as you are destructuring the array on call site anyway.
To make it more robust and less surprising, could we just pass `ageGroups` as a simple array? Perhaps defaulting to an empty array, if needed.

Reply:
I also don't really see why the `...ageGroups` is necessary. Why doesn't it work w/o `...`?

Reply:
The ... allows one to call the function like exportSimulation(result, "total", "80+"). Agreed its not necessary but it is more flexible. I've made the changes.